recipe_scrapers/donnahay.py

```diff
@@ -38,7 +38,7 @@ def site_name(self):
         return "Donna Hay"
 
     def title(self):
-        return self.soup.find("h1", class_="recipe-title__mobile").text
+        return self.soup.find("title").text.split("|")[0].strip()
```

Comment:
```suggestion
        html_title = self.soup.find("title")
        recipe_title, _, _ = html_title.text.rpartition("|")
        return recipe_title.strip()
```

Reply:
Ah, nope.. not quite correct.  `rpartition` would return an empty result when `|` is not found in the string.

Reply:
(updated/fixed to use `str.partition` instead)